core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/RegisterMinorWaitForParentViewModel.kt

```diff
@@ -0,0 +1,10 @@
+package com.ustadmobile.core.viewmodel
+
+data class RegisterMinorWaitForParentUiState(
+
+    val username: String = "",
+
+    val password: String = "",
+
+    val passwordVisible: Boolean = false,
```

Comment:
Please remove passwordVisible from the UiState class. The ViewModel won't manage this. It should be handled by using state in the composable, like how we handle the password field. This is because there is no logic connected to the password being visible or invisible.